SPECS/pg_bulkload-pg10.spec

```diff
@@ -54,8 +54,8 @@ install -m 755 lib/pg_bulkload.so              %{buildroot}%{_libdir}/pg_bulkloa
 install -m 644 lib/pg_bulkload.sql             %{buildroot}%{pg_contribdir}/pg_bulkload.sql
 install -m 644 lib/uninstall_pg_bulkload.sql   %{buildroot}%{pg_contribdir}/uninstall_pg_bulkload.sql
 install -m 644 lib/pg_bulkload.control         %{buildroot}%{pg_extensiondir}/pg_bulkload.control
-install -m 644 lib/pg_bulkload--1.0.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--1.0.sql
-install -m 644 lib/pg_bulkload--unpackaged--1.0.sql         %{buildroot}%{pg_extensiondir}/pg_bulkload--unpackaged--1.0.sql
+install -m 644 lib/pg_bulkload--%{version}.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--%{version}.sql
+install -m 644 lib/pg_bulkload--unpackaged--%{version}.sql         %{buildroot}%{pg_extensiondir}/pg_bulkload--unpackaged--%{version}.sql
```

Comment:
Do we need remove the lines too?

Reply:
Please check if you can create the rpms using the specs.

Reply:
No, we don't need these lines. So I removed its.
Then I tested create the rpms and it seems fine.

Reply:
Thanks!